DuckDuckGo/Onboarding/ContextualOnboarding/ViewHighlighter/ContextualOnboardingViewHighlighter.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ContextualOnboardingViewHighlighter.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import AppKit
+import Lottie
+
+enum ContextualOnboardingViewHighlighter {
```

Comment:
Maybe we can Just call it ViewPulsingHighlighter or something more generic?
Probably it can also love somewhere more generic?

Reply:
Changed to ViewHighlighter. For the moment I leave it in that folder since we’re using only for contextual onboarding